mackerel-plugin-redis/lib/redis.go

```diff
@@ -233,7 +233,8 @@ func (m RedisPlugin) GraphDefinition() map[string]mp.Graphs {
 			Unit:  "integer",
 			Metrics: []mp.Metrics{
 				{Name: "keys", Label: "Keys", Diff: false},
-				{Name: "expired", Label: "Expired Keys", Diff: false},
+				{Name: "expires", Label: "Expire Time Set Keys", Diff: false},
```

Comment:
I think `Keys with expiration` or `Keys with associated expire` would be better.

Reply:
or `Keys which expires`